src/anyvar/anyvar.py

```diff
@@ -83,10 +82,10 @@ def put_object(self, variation_object: VrsObject) -> Optional[str]:
         :return: Object digest if successful, None otherwise
         """
         try:
-            v = vrs_enref(variation_object, self.object_store)
+            id, v = vrs_enref(variation_object, self.object_store, True)
```

Comment:
`v` isn't being used anywhere, so we could just set to `_`

```suggestion
            id, _ = vrs_enref(variation_object, self.object_store, True)
```